src/components/web-monitoring-ui.jsx

```diff
@@ -44,7 +46,8 @@ export default class WebMonitoringUi extends React.Component {
     this.afterLogin = this.afterLogin.bind(this);
     this.logOut = this.logOut.bind(this);
     this.loadPages = this.loadPages.bind(this);
-    this.search = this.search.bind(this);
+    this.urlSearch = this.urlSearch.bind(this);
+    this.dateSearch = this.dateSearch.bind(this);
```

Comment:
This works, but the reason I had named the previous handler `search` and had it take a query *object* instead of just a URL was that I was thinking things would get way out of hand if we did something like this (after all, there are a lot of other fields we could search besides URL and date). `WebMonitoringUi.search()` was meant to take a query object with all the search fields set on it.